### PR TITLE
LPS-178861 | Show title of the section in the dataset views list page

### PR DESCRIPTION
**Pull request by andreluis2022**
**Description:**
Create tests - Product QA | Functional Automation - LPS-178861 Show title of the section in the dataset views list page

**JIRA Ticket:**
https://liferay.atlassian.net/browse/LPS-178861

### CODE REVIEW

**Comment by andreluis2022** [comment]
ci:test:frontend-data-set

**Comment by andreluis2022** [comment]
ci:stop

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by andreluis2022** [comment]
ci:test:frontend-data-set

**Comment by andreluis2022** [comment]
Hey @CarlosBrichete can you take a look in this test ?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSet.testcase [hunk 1]
```diff
@@ -306,6 +390,34 @@ definition {
...
+				key_name = "Test View");
+		}
+
+		task ("Then the title "Views" is present at the top of the list") {
+			AssertElementPresent(
+				key_title = "Views",
+				locator1 = "Header#H2_TITLE");
+		}
+	}
+
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSet.testcase:420]: I don't see the value on adding this test, you can remove it

**Review by CarlosBrichete — CHANGES_REQUESTED** [review]
LGTM, just remove the test and resolve conflicts with master. After that, you can forward it :)

**Pull request closed without merging**